### PR TITLE
New intro modal testing

### PR DESCRIPTION
**Pull request by sythel**
live on shiny2.gsscdev.com

Issue #200 relevant 

- check for previous runs before giving user option to load
- allows group mode to load either average or personal run
- bug fixes

things to review: button text addition/changes for Group load score modal

### CODE REVIEW

File: src/dams_mcda/server.R [hunk 1]
```diff
@@ -453,6 +440,91 @@ server <- function(input, output, session) {
...
+		if (session_mode == "individual"){
+			if (has_previous_session){
+				modal_div <- div(
+					HTML( "<h4>Are you loading scores from a previous session or creating a new run?</h4>"),
+					actionButton("uploadBtn", "Upload Scores From File"),
+					actionButton("loadIndivScoresBtn", paste0("Load Saved ", sapply(session_mode, capStringFirst), " Session")),
+					actionButton("startNewBtn", "New Run"),
+					HTML( "<h4>Instructions for Uploading</h4>\
+						Use this option only if you have done this activity before. Your input file should be in .CSV format, \
+						and your data should be organized in 9 rows (criteria header, dams) with 15 columns (dam names, decision criteria). Cells should be\
```
Comment (elbfox) [inline src/dams_mcda/server.R:464]:
```suggestion
						and your data should be organized in 9 rows (criteria header, dams) with 13 columns (dam names, decision criteria). Cells should be\
```

File: src/dams_mcda/server.R [hunk 2]
```diff
@@ -453,6 +440,91 @@ server <- function(input, output, session) {
...
+					)
+				)
+			}else{
+				modal_div <- div(
+					HTML( "<h4>Are you loading scores from a previous session or creating a new run?</h4>"),
+					actionButton("uploadBtn", "Upload Scores From File"),
+					actionButton("startNewBtn", "New Run"),
+					HTML( "<h4>Instructions for Uploading</h4>\
+						Use this option only if you have done this activity before. Your input file should be in .CSV format, \
+						and your data should be organized in 9 rows (criteria header, dams) with 15 columns (dam names, decision criteria). Cells should be\
```
Comment (elbfox) [inline src/dams_mcda/server.R:477]:
```suggestion
						and your data should be organized in 9 rows (criteria header, dams) with 13 columns (dam names, decision criteria). Cells should be\
```

File: src/dams_mcda/server.R [hunk 3]
```diff
@@ -453,6 +440,91 @@ server <- function(input, output, session) {
...
+			if (has_previous_session){
+				modal_div <- div(
+					HTML( "<h4>Are you loading scores from a previous session or creating a new run?</h4>"),
+					actionButton("uploadBtn", "Upload Scores From File"),
+					actionButton("loadIndivGroupScoresBtn", paste0("Load Saved Preferences")),
+					actionButton("loadGroupScoresBtn", paste0("Load Saved ", sapply(session_mode, capStringFirst), " Average Preferences")),
+					actionButton("startNewBtn", "New Run"),
+					HTML( "<h4>Instructions for Uploading</h4>\
+						Use this option only if you have done this activity before. Your input file should be in .CSV format, \
+						and your data should be organized in 9 rows (criteria header, dams) with 15 columns (dam names, decision criteria). Cells should be\
```
Comment (elbfox) [inline src/dams_mcda/server.R:494]:
```suggestion
						and your data should be organized in 9 rows (criteria header, dams) with 13 columns (dam names, decision criteria). Cells should be\
```

File: src/dams_mcda/server.R [hunk 4]
```diff
@@ -453,6 +440,91 @@ server <- function(input, output, session) {
...
+				)
+			}else{
+				modal_div <- div(
+					HTML( "<h4>Are you loading scores from a previous session or creating a new run?</h4>"),
+					actionButton("uploadBtn", "Upload Scores From File"),
+					actionButton("loadGroupScoresBtn", paste0("Load Saved ", sapply(session_mode, capStringFirst), " Average Preferences")),
+					actionButton("startNewBtn", "New Run"),
+					HTML( "<h4>Instructions for Uploading</h4>\
+						Use this option only if you have done this activity before. Your input file should be in .CSV format, \
+						and your data should be organized in 9 rows (criteria header, dams) with 15 columns (dam names, decision criteria). Cells should be\
```
Comment (elbfox) [inline src/dams_mcda/server.R:508]:
```suggestion
						and your data should be organized in 9 rows (criteria header, dams) with 13 columns (dam names, decision criteria). Cells should be\
```

**Comment by sythel** [comment]
fixed file upload cancel issue, fixed merge conflicts, merging

**Pull request merged**